src/dialog-options.js

```diff
@@ -4,5 +4,6 @@ export let dialogOptions = {
   startingZIndex: 1000,
   ignoreTransitions: false,
   rejectOnCancel: false,
-  yieldController: false
+  yieldController: false,
+  enableEscClose: true
```

Comment:
I think we should default to `false` initially, thoughts?  Since lock is default true this overrides that behavior by default.  I think we should make this conventionally false and configurably true.